Introduction/Introduction.md

```diff
@@ -0,0 +1,23 @@
+Kernel introduction.
+================================================================================
+
+The Begining
+--------------------------------------------------------------------------------
+Linux has began in the Operating system more important in the last years and many people use it in their lives commonly.
+But nobody like remember the history about the Linux kernel.
+
+It s now installed on millions of systems and used by home users and professionals alike for a wide range of tasks. From miniature embedded systems in wristwatches to massively parallel mainframes, the are countless way of exploiting linux productively.
+
+This the first part to undestand if you want to begin with this topics.
+Linux looks like any other Unix systems; indeed, UNIX compatiblity has a major design and goal of the Linux Kernel project and it is considered as the core of the Operating Systems.
```

Comment:
- any other system (not system<s>s</s>)
- I'd consider dropping the semicolon
- compatib**i**lity
- has **been** a major
- I think I know what you mean, but I would rather write 'design goal' than 'design and goal'.
- What do you mean by 'it is considered as the core of the OSes'?